ts/Dashboard/EditMode/EditContextMenu.ts

```diff
@@ -44,12 +44,15 @@ class EditContextMenu extends Menu {
     *
     * */
     constructor(
+        parentElement: HTMLElement,
+        options: EditContextMenu.Options,
         editMode: EditMode,
-        options?: EditContextMenu.Options|undefined
+        parent?: any
     ) {
         super(
             editMode.dashboard.container,
-            merge(EditContextMenu.defaultOptions, options || {})
+            merge(EditContextMenu.defaultOptions, options || {}),
+            editMode
```

Comment:
Why do we need it here?

Reply:
We don't exactly need it here but I'm trying to unify the constructors of each class that extends the `Menu` class.